libs/text-splitters/langchain_text_splitters/character.py

```diff
@@ -577,3 +577,61 @@ def get_separators_for_language(language: Language) -> List[str]:
                 f"Language {language} is not supported! "
                 f"Please choose from {list(Language)}"
             )
+
+
+class SemanticTiktokenTextSplitter(TextSplitter):
+    """Splitting text that looks at characters."""
+
+    def __init__(
+        self,
+        max_characters: int = 200,
+        model_name: str = "gpt-3.5-turbo",
+        trim_chunks: bool = False,
+        **kwargs: Any,
+    ) -> None:
+        """Create a new TextSplitter."""
+        super().__init__(**kwargs)
+        self.max_characters = max_characters
+        self.trim_chunks = trim_chunks
+        self.model_name = model_name
+
+    def split_text(self, text: str) -> List[str]:
+        """Split incoming text and return chunks."""
+        # First we naively split the large input into a bunch of smaller ones.
+        try:
+            from semantic_text_splitter import TiktokenTextSplitter
+        except ImportError as e:
+            raise ImportError(
+                "Unable to import from semantic-text-splitter. Please specify "
+                "text_splitter or install semantic-text-splitter with "
```

Comment:
what does it mean to specify a text_splitter in the error message?

Reply:
Updated error message